### ISSUE TITLE
gamesandstuff windows launcher doesn't have window borders

### ISSUE THREAD

**Issue by Hoot679** [letsgoaway-games/gamesandstuff#1]
upon opening the gamesandstuff windows launcher, the window has no borders
the text is also kinda cut off
(https://user-images.githubusercontent.com/70825884/200106699-51faad61-6287-4d4c-9ee9-b3b7953f2a66.PNG)

**Comment by letsgoaway-games** [comment]
the borderless window is intentional, the text being cut off is not....

**Comment by letsgoaway-games** [comment]
ill try to fix, seems like something to do with the css styling

**Comment by letsgoaway-games** [comment]
I have been able to reproduce the issue, it seems like it happens when your scale under display settings is higher than 100%
![image](https://user-images.githubusercontent.com/68365423/200107696-ca5aa927-5620-4494-9c46-9487cf917aea.png)
ill try figure out a patch and send a hopefull fixed version to you

**Comment by letsgoaway-games** [comment]
@Hoot679 try this and send a screenshot [https://gamesandstuffdl.letsgoaway.repl.co](https://gamesandstuffdl.letsgoaway.repl.co)

**Comment by Hoot679** [comment]
It seems to be fixed once the resolution is set to 100%

**Comment by Hoot679** [comment]
![works](https://user-images.githubusercontent.com/70825884/200119497-16a7d098-61ee-4345-8ce9-b3e116bf2b40.PNG)

**Comment by letsgoaway-games** [comment]
yes, if you can live with squinting to view things. ill keep this issue open for now so i know it is still an issue

**Comment by letsgoaway-games** [comment]
especially considering monitor sizing is getting bigger and scale options are more usefull

**Comment by Hoot679** [comment]
it also is ok with 125% scaling, but the open in app mode button is cut off
<img width="507" alt="looks like this" src="https://user-images.githubusercontent.com/70825884/200151840-c3a1e966-153a-4bba-91dc-24a0a320a7de.PNG">

**Comment by letsgoaway-games** [comment]
hmmm

**Comment by Hoot679** [comment]
in my tv on 125%, it works fine
